src/createViews.js

```diff
@@ -0,0 +1,128 @@
+class CreateViews {
+  constructor() {
+    this.peepContainer = document.querySelector("#basePeep");
+    this.container = document.querySelector("#peepContainer");
+    this.timeFrames = {
+      day: 24 * 60 * 60,
+      hour: 60 * 60,
+      minunte: 60,
+      second: 1,
+    };
+    this.lastPeepCounter = 0;
+  }
+  add(peep, owner) {
+    let peepBlock = this.peepContainer.cloneNode(true);
+    peepBlock.removeAttribute("id");
+    this.owner = owner;
+    this.peep = peep;
+    this.fillFields(peepBlock);
+    this.container.insertAdjacentElement("beforeend", peepBlock);
```

Comment:
Instead of cloning a node from the page, you should just recreate a node in here. Either using createElement + appendChild or using innerHTML = `<div> etc.</div>` or even a mix of both.

Reply:
Yes will not do it again